src/test/java/red/softn/npedidos/service/MenuServiceTestUtil.java

```diff
@@ -0,0 +1,33 @@
+package red.softn.npedidos.service;
+
+import net.datafaker.Faker;
+import red.softn.npedidos.CrudServiceTestUtil;
+import red.softn.npedidos.entity.Menu;
+import red.softn.npedidos.request.MenuRequest;
+import red.softn.npedidos.response.MenuResponse;
+
+class MenuServiceTestUtil extends CrudServiceTestUtil<MenuRequest, MenuResponse, Menu, Integer> {
+    
+    public MenuServiceTestUtil(Faker faker) {
+        super(faker);

```

Comment:
Please update the branch with the master changes.
After updating the branch, remove the `Faker` parameter.

Reply:
@nmarulo updated the branch. PTAL